dafni_cli/model.py

```diff
@@ -105,27 +102,53 @@ def filter_by_date(self, key: str, date: str) -> bool:
         else:
             raise KeyError("Key should be CREATION or PUBLICATION")
 
-    def output_model_details(self):
+    def output_model_details(self, long=False):
```

Comment:
update to long: bool = False

Reply:
When you do dafni.exe get models the output doesnt always have a line seperating the individual models. However it does when you do --long which actually makes it alot easier to read. Might suggest adding a line break in between the models if you dont give --long as well?